### PR TITLE
Superfund frontend alternate

### PR DESCRIPTION
**Pull request by gusteran**
Updates the UI of the map to that of the demo

- Replaces blue background with underline
- Adds underline to pollutants to make it more clear that they are clickable
- Add the link to the EPA site for each superfund site infowindow

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -79,11 +79,11 @@
       <scope>test</scope>
     </dependency>
 
-    <dependency>
+    <!-- <dependency>
```
Comment (lazau-g) [inline pom.xml:82]: just delete

File: pom.xml [hunk 2]
```diff
@@ -159,11 +159,11 @@
         </configuration>
       </plugin>
 
-      <plugin>
+      <!-- <plugin>
```
Comment (lazau-g) [inline pom.xml:162]: delete

File: src/main/webapp/map.js [hunk 3]
```diff
@@ -134,24 +142,27 @@ function addSuperfundMarkers(areaType, areaCode){
...
-function iconUrl(score){
-	var url = "https://maps.google.com/mapfiles/ms/icons/";
-	if(score < low_danger) url+= "green";
-	else if(score < medium_danger) url += "yellow";
-	else if (score < high_danger) url +="orange";
-    else url +="red";
-	url += "-dot.png";
-	return url;
-}
+// const low_danger = 25;
```
Comment (lazau-g) [inline src/main/webapp/map.js:145]: delete

**Comment by gusteran** [comment]
That'll be fixed for real in this coming small PR for the branch deploy, that was a quick bugfix

**Pull request merged**